testeng/jobs/backupJenkins.groovy

```diff
@@ -69,18 +69,18 @@ secretMap.each { jobConfigs ->
         // during snapshotting
         configure { project ->
             project / buildWrappers << 'hudson.plugins.execution.exclusive.ExclusiveBuildWrapper' {
-                skipWaitOnRunningJobs false
+                skipWaitOnRunningJobs true
```

Comment:
nit: It'd be worth a comment here since this requires a bit of mental gymnastics. e.g. Don't wait for running jobs to finish.